### PR TITLE
cli: use rich.table Table to auto-format tables

### PR DESCRIPTION
**Pull request by willcl-ark**

### CODE REVIEW

**Comment by willcl-ark** [comment]
Switched to use `rich.table Table` as we already have that dep, so no need for tabulate.

Kept server returning a dict and did table processing inside CLI, which kept the tests happier (and made more sense).

screenies

image

image

image

File: src/warnet/warnet.py [hunk 1]
```diff
@@ -55,6 +55,39 @@ def __str__(self) -> str:
             f"Tanks:\n{tanks_str}"
         )
 
+    def _wn_repr(self) -> dict:
```
Comment (pinheadmz) [inline src/warnet/warnet.py]: what does "_wn_repr" mean?

**Review by pinheadmz — COMMENTED** [review]
ACK 
looks good nice!
one optional nit, function name is confusing

**Comment by willcl-ark** [comment]
> ACK 
> 
> looks good nice!
> 
> one optional nit, function name is confusing

Which, `print_repr`?

File: src/warnet/warnet.py (hunk 1 above)
Comment (pinheadmz) [reply src/warnet/warnet.py]: @willcl-ark

Comment (willcl-ark) [reply src/warnet/warnet.py]: Warnet Representation

Its a python thing: https://docs.python.org/3/library/functions.html#repr

But I can change if its blocking?

Comment (willcl-ark) [reply src/warnet/warnet.py]: Actually, now that it returns a `dict` I don't think it's a "real" repr :P

In a mid-edit version I had this returning the `tabulate` tables as strings, but no longer...

Comment (willcl-ark) [reply src/warnet/warnet.py]: Updated in 807dab37497b535470c16dd50c3574cacc28b097

**Review by pinheadmz — APPROVED** [review]
|review|
|-----|
| ACK |

**Pull request merged**